README.md

```diff
@@ -0,0 +1,59 @@
+# KNWRobot - Robotics Control Library for SMU KNW2300
+Controling Arduino: MEGA 2560
+
+Author: Alexandria Hancock
+
+## Setup
+Before using the library, you must first setup your laptop to be able to compile code written using Arduino C++. You will then need to add additional libraries that the code uses behind the scenes (they are included in this repo). Follow these steps:
+
+1) Download the [Arduino IDE](https://www.arduino.cc/en/Main/Software) for your operating system. Go through the installation steps.
+	* Windows: proceed with the default installation options.
+	* Mac OSX: drag and drop the application into your Applications directory (most often `/Applications`).
+
+After finishing the installation, open the Arduino IDE. Continue selecting default options until you see a text editor with some empty functions. Close the application for now.
+
+2) [Download the latest release of this repository](https://github.com/SMUKNW2300/arduino-library/releases). Unzip the files onto your computer.
+
+3) Copy over the contents of the `lib` folder in this repo into the `libraries` folder of your Arduino installation.
+	* Windows: Open File Explorer, go to `My Documents`. You should see an `Arduino` folder. Within that folder is a `libraries` folder (create one if it doesn't exist). Copy the directories in `lib` into the `libraries` folder.
+	* Mac: Open finder, go to your Documents directory (/Users/[username]/Documents). You should see an `Arduino` folder. Within that folder is a `libraries` folder (create one if it doesn't exist). Copy the directories in `lib` into the `libraries` folder.
+
+4) Copy the `knw` folder (located in `src`) into the `libraries` folder you opened in step 3.
+
+5) Open the Arduino IDE again. At the top of the file, add the following line:
+
+```cpp
+#include <KNWRobot.h>
+```
+
+6) At the top left of the Arduino IDE, click the checkmark icon (Verify). This will compile the code. If you followed the steps correctly, you will see a message like `Done compiling`. If a step was missed, you will see error messages. If this happens, please ask a TA for help.
+
```

Comment:
We might also want to give a nod to the install video on Canvas.  <aside> I wish the videos were on something like youtube or vimeo so they were linkable between semesters</aside>

Reply:
I can upload the install video to YouTube as unlisted  (or look at embedded playing from Box) and add a link to that here.

Reply:
I wouldn't worry about it for now.  We can edit later.